libs/environment/src/components/node-switcher/node-switcher.tsx

```diff
@@ -139,31 +152,35 @@ export const NodeSwitcher = ({ closeDialog }: { closeDialog: () => void }) => {
 };
 
 interface CustomRowWrapperProps {
-  inputText: string;
-  setInputText: (text: string) => void;
-  nodes: string[];
+  graphQLApiUrl: string;
```

Comment:
See my main comment, but I don't think we need to change anything in the node switcher, if we continue to use VEGA_URL without the gql path